test/src/e2e/multisig.test.ts

```diff
@@ -113,8 +149,8 @@ describe("Multisig", function() {
                 );
 
             const hash = await node.sendAssetTransaction(transfer);
-            expect(await node.sdk.rpc.chain.containsTransaction(hash)).be.true;
-            expect(await node.sdk.rpc.chain.getTransaction(hash)).not.null;
+            expect(await chain.containsTransaction(hash)).be.true;
```

Comment:
Why do we need these changes?

Reply:
I think these changes makes the code looks better. Is this sort of refactoring not desirable?

Reply:
Adding a method to `node` would be good.
But I think it doesn't enhance the readability because the reader must check what `chain` means.

Reply:
Ok. This change has been reverted.